src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/data.html.twig

```diff
@@ -23,4 +23,14 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 
-{{ row[column.options.field] }}
+{% if 'center' == column.options.align %}
```

Comment:
The column must not be aware of his design => NO GO.

Reply:
you mean add styles using javascript?

Reply:
only add css classes :)

Reply:
no, using javascript you add class to the column(s) you need to center, you don't rely to column to do it and you remove this `align` property from column options

Reply:
> and i need to center only one

Don't we have a way to override only 1 column of a Grid? https://github.com/PrestaShop/prestafony-project/issues/40

If guess you ma work on making the templating management more powerful first, before designing this column 👍